xilem/src/view/mod.rs

```diff
@@ -50,3 +50,42 @@ pub use portal::*;
 
 mod zstack;
 pub use zstack::*;
+
+/// An extension trait, to allow common transformations of the views transform.
+pub trait Transformable: Sized {
+    fn transform_mut(&mut self) -> &mut crate::Affine;
+
+    #[must_use]
+    fn rotate(mut self, radians: f64) -> Self {
+        let transform = self.transform_mut();
+        *transform = transform.then_rotate(radians);
+        self
+    }
+
+    #[must_use]
+    fn scale(mut self, uniform: f64) -> Self {
+        let transform = self.transform_mut();
+        *transform = transform.then_scale(uniform);
+        self
+    }
+
+    #[must_use]
+    fn scale_non_uniform(mut self, x: f64, y: f64) -> Self {
+        let transform = self.transform_mut();
+        *transform = transform.then_scale_non_uniform(x, y);
+        self
+    }
+
+    #[must_use]
+    fn translate(mut self, v: impl Into<crate::Vec2>) -> Self {
+        let transform = self.transform_mut();
+        *transform = transform.then_translate(v.into());
+        self
+    }
+
+    #[must_use]
+    fn transform(mut self, v: impl Into<crate::Affine>) -> Self {
```

Comment:
The naming here is a little bit unfortunate, as there's already `transform_mut`, I'm not sure of a better name for setting a general transform.